lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -117,12 +117,21 @@
     name: all
     target: /tmp/dump.sql
 
-- name: Import file.sql similar to mysql -u <username> -p <password> < hostname.sql
+# Basic import of sql script file
```

Comment:
For conciseness of the examples, maybe should we keep only 1 example with multiple option enabled / shown.
I'd keep only your example